en-US/Players_Guide/04_Adventuring/Exploration_Environment.md

```diff
@@ -0,0 +1,108 @@
+The rules in this section cover some of the most important ways in which adventurers interact with the environment in such places.
+
+##### Falling
+
+A fall from a great height is one of the most common hazards facing an adventurer.
+At the end of a fall, a creature takes `1d6 bludgeoning damage for every 10 feet it fell`, to a `maximum of 20d6`.
+The creature lands prone, unless it avoids taking damage from the fall.
+
+##### Suffocating
+
+A creature can hold its breath for a number of minutes equal to `1 + its Constitution modifier` (minimum of 30 seconds).
+
+When a creature runs out of breath or is choking, it can survive for a number of rounds equal to `its Constitution modifier` (minimum of 1 round).
+At the start of its next turn, it drops to 0 hit points and is dying, and it can’t regain hit points or be stabilized until it can breathe again.
+
+For example, a creature with a Constitution of 14 can hold its breath for 3 minutes.
+If it starts suffocating, it has 2 rounds to reach air before it drops to 0 hit points.
+
+##### Senses and Awareness
+
+The most fundamental tasks of adventuring—detecting danger, finding hidden objects, hitting a target, or casting a spell—depend on your ability to [**sense**](#Exploration_Environment_sense) or [**pinpoint**](#Exploration_Environment_pinpoint) others.
+Darkness and similar effects that obscure sensory cues can hinder these actions.
+
+A given area might be lightly or heavily obscured.
+In a **lightly obscured** area, such as dim light, patchy fog, or moderate foliage, creatures have disadvantage on Wisdom (Perception) checks that rely on sight.
+
+A **heavily obscured** area—such as darkness, opaque fog, or dense foliage—blocks vision entirely.
+A creature effectively suffers from the [blinded condition](#Conditions_blinded) (see ["Conditions"](#Conditions_conditions)) when trying to see something in that area.
+
+##### Pinpoint {#Exploration_Environment_pinpoint}
+
+A creature can pinpoint a target’s location only if the target provides a relevant sensory cue—visibility for sight, vibrations for tremorsense, odor for scent, and so on.
+
+Precise senses (sight, blindsight, tremorsense\*, truesight) allow a creature to pinpoint a target’s exact location within the sense’s range.
+
+Imprecise senses (hearing, scent) allow pinpointing only within 15 feet; beyond 15 feet, the creature can attempt a Wisdom (Perception) check (`DC = 10 + 1 per 5 feet to the target`) to pinpoint within 30 feet.
+
+Environmental factors such as noise or wind can reduce a sense’s effectiveness and may obscure the area for that sense.
+
+If an effect depends on a creature pinpointing you using an imprecise sense, use its passive Wisdom (Perception) to determine whether it succeeds.
+\
+\
+<mark>\* See the chapter _Monster Information_ in _Free-Range Monsters_ for more information on tremorsense.</mark>
```

Comment:
Since tremorsense isn't explained in the Player's Guide (and there are few ways in which a player character could ever have tremorsense), I added this footnote.
Obviously we could also add an explanation of tremorsense instead.